### ISSUE TITLE
Role change in workspace cause anoying blink in GUI

### ISSUE THREAD

**Issue by touilleMan** [Scille/parsec-cloud#910]
Changing when the role on a workspace is changed (or when the workspace is re-encrypted), the corresponding user gets his gui workspace view to blink in a anoying way.

**Comment by vamonte** [comment]
I don't seem able to reproduce this issue.
To reproduce it, I use this test case:

1.  I have two tabs open. The first one with the alice account and the second one with the bob account. 

2.  One workspace is shared by alice to bob.

3.  alice change the bob's role on this workspace

4. there are not blink on the bob's tab.

**Does the blink happen only one time and I'm not able to see him because I use two tabs instead of two windows?**

**Comment by touilleMan** [comment]
Yup, I guess the blink occurses right at the time the message from the backend is processed by the client. So with a backend running on local I guess the lag is really low so the processing is already over by the time you select the second tab ;-)

Note you can run multiple instances of parsec  on the same computer with the `run_testenv` script (https://github.com/Scille/parsec-cloud/blob/master/docs/development.rst#run-parsec-in-local)

**Comment by vamonte** [comment]
@touilleMan as you explained, I tried to run two instances of parsec on my laptop. Same results.
The new test case is:

1. `. ./tests/scripts/run_testenv.sh && parsec core gui` in a first shell

2.  `. ./tests/scripts/run_testenv.sh --empty && parsec core gui` in a second shell

3. In the first instance, I create a new device for the Alice user.

4. I use the second instance to register the new device and to connect to it.

5.  In the first instance,  I change the alice's role with the bob account

6. there are not blink on the second instance.

As @Max-7 said on slack, I tried to add some latency with the command `tc qdisc add dev lo root handle 1:0 netem delay 400msec` but there are not always blink.
